actionview/test/template/url_helper_test.rb

```diff
@@ -770,6 +770,70 @@ def test_sms_does_not_modify_html_options_hash
     assert_equal({ class: "special" }, options)
   end
 
+  def test_phone_to
+    assert_dom_equal %{<a href="tel:1234567890">1234567890</a>},
+      phone_to("1234567890")
+    assert_dom_equal %{<a href="tel:1234567890">Bob</a>},
+      phone_to("1234567890", "Bob")
+    assert_dom_equal(
+      %{<a class="phoner" href="tel:1234567890">Bob</a>},
+      phone_to("1234567890", "Bob", "class" => "phoner")
+    )
+    assert_equal phone_to("1234567890", "Bob", "class" => "admin"),
+                 phone_to("1234567890", "Bob", class: "admin")
+  end
+
+  def test_phone_to_with_options
+    assert_dom_equal(
+      %{<a class="example-class" href="tel:+011234567890">Phone</a>},
+      phone_to("1234567890", "Phone", class: "example-class", country_code: "01")
+    )
+
+    assert_dom_equal(
+      %{<a href="tel:+011234567890">Phone</a>},
+      phone_to("1234567890", "Phone", country_code: "01")
+    )
+  end
+
+  def test_phone_with_img
```

Comment:
The test description should probably say `phone_to` and not `phone` as we are testing `phone_to` here. Similarly, you might need to change it other tests you have added here.

```suggestion
  def test_phone_to_with_img
```

Reply:
Oh. It was merged before I comment. Maybe followup of team members agree.

Reply:
I think it is a good idea

Reply:
ok.